app/models/application_record.rb

```diff
@@ -77,6 +77,23 @@ def pending?
     !reviewed
   end
 
+  def save_for_later(date = nil, note = nil, mail = false)
+    update_attributes(saved_for_later: true,
+                      date_for_later: date,
+                      note_for_later: note)
+    JobappsMailer.send_note_for_later(self).deliver_now if mail
+  end
+
+  def unsave
```

Comment:
I honestly do not know why I didn't choose 'move_to_dashboard' given that there's a cron job of almost the same name.